cmd/cluster-olm-operator/main.go

```diff
@@ -32,10 +33,20 @@ func main() {
 }
 
 func newRootCommand() *cobra.Command {
+	var versionFlag bool
+
 	cmd := &cobra.Command{
 		Use:   "cluster-olm-operator",
 		Short: "OpenShift Cluster OLM Operator",
+		Run: func(cmd *cobra.Command, args []string) {
+			if versionFlag {
+				fmt.Println(version.Get())
+				os.Exit(0)
+			}
+			cmd.Help()
```

Comment:
Linter is upset about no error check here

Reply:
Thanks! Updated it.